core/primitives/src/version.rs

```diff
@@ -52,6 +52,8 @@ pub const DELETE_KEY_STORAGE_USAGE_PROTOCOL_VERSION: ProtocolVersion = 40;
 
 pub const SHARD_CHUNK_HEADER_UPGRADE_VERSION: ProtocolVersion = 41;
 
+pub const BLOCK_ORDINAL_UPGRADE_VERSION: ProtocolVersion = 45;
```

Comment:
@bowenwang1996 what is the current process for the protocol features? Should all of them go through nightly? This version 45 while the stable is 41, and the latest nightly is 47 looks off to me.

Reply:
Added the feature to nightly.